docs/docs/addons-dom-factories.md

```diff
@@ -0,0 +1,34 @@
+---
+id: dom-factories
+title: DOM Factories
+permalink: docs/dom-factories.html
+layout: docs
+category: Add-Ons
```

Comment:
We removed the 'Add-Ons' section of the docs from the main navigation when we deprecated most of them in v15.5. (https://github.com/facebook/react/pull/9359) I think it's worth having this documentation - maybe we could put it in the `README` for the new separate module?